okta/komand_okta/actions/get_factors/action.py

```diff
@@ -0,0 +1,31 @@
+import komand
+from .schema import GetFactorsInput, GetFactorsOutput, Input, Output, Component
+# Custom imports below
+import requests
+
+
+class GetFactors(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name='get_factors',
+            description=Component.DESCRIPTION,
+            input=GetFactorsInput(),
+            output=GetFactorsOutput())
+
+    def run(self, params={}):
+        self.logger.info("Getting factors %s", params)
+        user_id = params.get(Input.USER_ID)
+        if not user_id:
+            raise ValueError('user_id is required')
```

Comment:
Use PluginException for any error messages: https://komand.github.io/python/error_handling.html#plugin-exceptions

Reply:
👍